### DISCUSSION TITLE
deploy multiple apps to different vercel projects

### DISCUSSION THREAD

**Discussion by salzhrani** [nrwl/nx#6275] [General]
The documentation explains how deploy a single app. Also, it recommends agains changing the root folder. how can we configure nx/vercel to   to deploy multiple apps to different projects properly and without triggering builds for other projects.  anyone who have done this please chime in.

**Comment by vincent-thomas** [comment]
I have this problem now aswell!

**Comment by coolbii** [comment]
any update ?
the vercel official doc is too simplified to understand what to configure for developers

**Comment by johnitvn** [comment]
I also had the same problem. I have searched many documents but vercel does not clearly explain this issue

**Reply by coolbii** [reply]
Ban neu can gap thi inbox minh nha  : >

**Comment by coolbii** [comment]
I finally figured out the way to combine with yarn workspace 

- frontend (nextjs)  
- backend (drizzle orm, express, graphql-yoga)
- shared


And the combination with Nx monorepos didn't work,  for the sake that Vercel is not really well adapted to the nx alias import


I will organize my work and post it here later for you guys
